nasa-gibs/worldview
# Add 2 Day/Night Band NOAA-20 layers

## Description

Fixes #WV-3068.

Added two Day/Night band layers from VIIRS/NOAA-20

## How To Test

1. git checkout wv-3068-noaa20-dnb
2. npm run build && npm start
3. View [this URL](http://localhost:3000/?v=-295.28403324353263,-150.93617883707057,232.9081788336875,108.19351812780454&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,VIIRS_NOAA20_DayNightBand_AtSensor_M15,VIIRS_NOAA20_DayNightBand_At_Sensor_Radiance,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2024-04-01-T17%3A14%3A44Z). Check to see if the imagery loads and that you can take an image snapshot
4. Go to "Add Layers", check that you can find the layers in Layer Picker and that the preview images are there


## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
